presto-main/src/main/java/com/facebook/presto/execution/TaskManagerConfig.java

```diff
@@ -77,6 +77,8 @@
 
     private boolean legacyLifespanCompletionCondition;
 
+    private boolean infoUpdateExponentialWait;
```

Comment:
nit: What about `useIncresaedWaitForInfoUpdate` ? ditto for session property names, etc.